README.md

```diff
@@ -2,18 +2,17 @@
 
 This gem is simply an abstraction for the various functions that are performed within the `spec/spec_helper_acceptance.rb` files across the modules to standardise how these are implemented.
 
-### `install_module_on`
+### `install_module`
+This will call `install_module_on` on the hosts with role 'master'. If there are none, the module will be install on all hosts with the role 'agent', again, if there are none, the module will be installed on all hosts.
 
-This will install the module under test on the specified host using the local source
+### `install_module_on`
+This will install the module under test on the specified host using the local source. The module name will be derived from the name property of the module's metadata.json file, assuming it is in format author-modulename.
```

Comment:
The documentation could also do with examples. People love copy&pasting working code ;-)